client/app/templates/application.hbs

```diff
@@ -1,22 +1,87 @@
-<h1>Project Athena</h1>
-<table>
-    <thead> 
-        <th>Game Number</th> 
-        <th>Game Time</th> 
-        <th>Rank</th> 
-        <th>Outcome</th> 
-    </thead> 
-    <tbody> 
-        {{#each model.games as |game|}}
-        <tr>
-          <td>{{game.id}}</td>  
-          <td>{{game.createdAt}}</td> 
-          <td>{{game.rank}}</td> 
-          <td>{{game.outcome}}</td>
-        </tr> 
-        {{/each}} 
-    </tbody> 
-</table> 
+<nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>          
+          <a class="navbar-brand" href="#">Athena</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">Dashboard</a></li>
+            <li><a href="#">Profile</a></li>
+            <li><a href="#">Login via Blizzard</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
 
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-offset-2 col-sm-8 col-md-8 main">
+          <h2>Add New Game</h2>
+          <form class="form-inline">
+            <div class="form-group">
+              <label for="rank">Rank</label>
+              <input type="rank" class="form-control" id="rank" size="4" maxlength="4">
+            </div>
+            {{#bs-button-group value=outcome type="radio" onChange=(action (mut outcome)) as |bg|}}
+              {{#bg.button type="default" value='W'}}Win{{/bg.button}}
+              {{#bg.button type="default" value='L'}}Loss{{/bg.button}}
+              {{#bg.button type="default" value='D'}}Draw{{/bg.button}}
+            {{/bs-button-group}}
+            <button type="submit" class="btn btn-default">Save</button>
+          </form>
+          <br>
+          <h2 class="page-header">Dashboard</h2>
```

Comment:
they will get moved to a component in the next feature when I move everything to components - we discussed this. I don't want to delete the code right away when we will need it in the future.